src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.shared.cs

```diff
@@ -7,7 +7,7 @@ namespace Xamarin.CommunityToolkit.UI.Views
 {
 	class SnackBar
```

Comment:
instead of shared, can you rename this file to `SnackBar.netstandard.cs`?

Reply:
@pictos if we go this way, then I believe we should rename all the other files (Effects, Views...) with a *.share.cs name, right ?

Reply:
If they throw exceptions yes. If that is valid code should have the suffix `shared`

Reply:
Fixed in https://github.com/xamarin/XamarinCommunityToolkit/pull/846/commits/7ddc949e8bb97cbf0b8265322df42ebd3265ff1c